receiver/splunkenterprisereceiver/metadata.yaml

```diff
@@ -30,81 +36,206 @@ metrics:
+    unit: '{%}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.aggregation.queue.ratio:
+    enabled: true
+    description: Gauge tracking the average indexer aggregation queue ration (%). *Note:** Search is best run against a Cluster Manager.
+    unit: '{%}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]  
+  splunk.typing.queue.ratio:
+    enabled: true
+    description: Gauge tracking the average indexer typing queue ration (%). *Note:** Search is best run against a Cluster Manager.
+    unit: '{%}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]  
+  splunk.indexer.queue.ratio:
+    enabled: true
+    description: Gauge tracking the average indexer index queue ration (%). *Note:** Search is best run against a Cluster Manager.
+    unit: '{%}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.scheduler.average.run.time:
+    enabled: true
+    description: Gauge tracking the average runtime of scheduled searches
+    unit: '{ms}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.indexer.raw.write.seconds:
+    enabled: true
+    description: Gauge tracking the number of raw write seconds per instance
+    unit: '{s}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.indexer.cpu.seconds:
+    enabled: true
+    description: Gauge tracking the number of indexing process cpu seconds per instance
+    unit: '{s}'
+    gauge:
+      value_type: double 
+    attributes: [splunk.host]
+  splunk.io.average.iops:
+    enabled: true
+    description: Gauge tracking the average IOPs used per instance
+    unit: '{iops}'
+    gauge:
+      value_type: int 
+    attributes: [splunk.host]
+  splunk.buckets.searchable.status:
+    enabled: true
+    description: Gauge tracking the number of buckets and their searchable status. *Note:** Search is best run against a Cluster Manager.
+    unit: '{count}'
+    gauge:
+      value_type: int 
+    attributes: [splunk.host, splunk.indexer.searchable]
+  splunk.indexes.bucket.count:
+    enabled: true
+    description: Gauge tracking the indexes and their bucket counts. *Note:** Search is best run against a Cluster Manager.
+    unit: '{count}'
+    gauge:
+      value_type: int 
+    attributes: [splunk.index.name]
+  splunk.indexes.size:
+    enabled: true
+    description: Gauge tracking the indexes and their total size (gb). *Note:** Search is best run against a Cluster Manager.
+    unit: Gb
+    gauge:
+      value_type: double 
+    attributes: [splunk.index.name]
+  splunk.indexes.avg.size:
+    enabled: true
+    description: Gauge tracking the indexes and their average size (gb). *Note:** Search is best run against a Cluster Manager.
+    unit: Gb
+    gauge:
+      value_type: double 
+    attributes: [splunk.index.name]
+  splunk.indexes.avg.usage:
+    enabled: true
```

Comment:
you're adding those metrics as enabled by default, so please make sure to change your changelog to "breaking".

Reply:
Will do on these! Thanks for pointing this out!

Receiver isn't "wired in" yet so wasn't sure if `breaking` was appropriate.